dashboard/app/models/script.rb

```diff
@@ -1635,40 +1622,27 @@ def summarize_i18n_for_copy(new_name, new_course_version)
     data['lessons'] = {}
     lessons.each do |lesson|
       lesson_data = {
-        'key' => lesson.key,
```

Comment:
@bethanyaconnor , was there any special reason you included the key when you copied lesson translations? It didn't exist for any scripts before 2021. these are the scripts I found them on:
```
ai-ethics-2021
coursea-2021
courseb-2021
coursec-2021
coursed-2021
coursee-2021
coursef-2021
csa6-2022
csa7-2022
csa8-2022
csd1-2021
csd1-2022
csd2-2021
csd2-2022
csd3-2022
csd4-preview-2021
csd5-2022
csd5-preview-2021
csd7-2022
csp1-2021
csp1-2022
csp10-2021
csp2-2021
csp2-2022
csp3-2021
csp3-2022
csp4-2021
csp4-2022
csp5-2021
csp5-2022
csp6-2021
csp6-2022
csp7-2021
csp7-2022
csp8-2021
csp9-2021
csp9-2022
express-2021
express-2022
hello-world-animals-2021
hello-world-emoji-2021
hello-world-food-2021
hello-world-retro-2021
k5-onlinepd-2021
k5-onlinepd-2022
kodea-pd-2021
poem-art-2021
pre-express-2021
pre-express-2022
self-paced-pl-csd2-2022
vpl-csd-2021
vpl-csd-2022
vpl-csp-2021
vpl-csp-2022
```

Reply:
It looks like that line was added in https://github.com/code-dot-org/code-dot-org/pull/36020/files#diff-22769ab80ff34eaf4aadc1dc87e1b9bdad8b445cec233c9a5a1d2bcc5a7a7978, which was before my time on cplat, so I can't provide background, sorry

Reply:
Oops, sorry to jump to conclusions Bethany. At the link you shared, it looks like a copy-paste error to me -- we were trying to populate the new lesson.key field from lesson.name, and it looks like we incorrectly also added a translation for the key at the same time, without a scenario where we'd need to ever show that translated value to the user. So I think this line is safe to remove.

Reply:
No worries -- I just didn't remember writing this :) I agree that it looks it can be removed. We never want to use translated unique identifiers anyway so it's better to not even give ourselves the option